src/index.js

```diff
@@ -78,3 +80,6 @@ module.exports.clearPermissions = clearPermissions
 module.exports.getPermissions = getPermissions
 module.exports.PermissionSet = PermissionSet
 module.exports.Authorization = Authorization
+
+// Export all the acl-related constants and modes at the top (index) level
+Object.assign(module.exports, aclModes.acl)
```

Comment:
Good point. I agree that `solid.acl.modes.EVERYONE` is a bit inconvenient. I still wonder though if there's a way that we can separate the concept of "access control functions/classes" from "access control constants". One approach would be `solid.aclModes.EVERYONE`.

Another view I have is that I really like that we're splitting apart the solid client package into smaller packages, and one philosophy to take would be to not re-export, and instead encourage developers to import directly from the `solid-permissions` library itself. This has the benefit of us only needing to maintain one set of solid permissions APIs (the interface exposed by `solid-permissions` rather than two (`solid-permissions` and the redundant exports in `solid-client`). I think of this as a sort of API normalization.

I won't force my opinion on this PR - but in the long term I would like to see us normalize our APIs.